AdyenActions/Actions/OXXOVoucherAction.swift

```diff
@@ -0,0 +1,37 @@
+//
+// Copyright (c) 2021 Adyen N.V.
+//
+// This file is open source and available under the MIT license. See the LICENSE file for more info.
+//
+
+import Foundation
+
+/// Describes an action in which an OXXO voucher is presented to the shopper.
+public class OXXOVoucherAction: GenericVoucherAction, DownloadableVoucher {
+
+    /// This reference is a short version of the barcode number of the voucher.
+    public let alternativeReference: String
+    
+    /// The reference to uniquely identify a payment.
+    /// This reference is used in all communication with you about the payment status.
```

Comment:
Maybe 'you' is a bit informal?

Reply:
in docs we use `you` to refer to the merchant, so its kind of a convention.